arm-relay/2016-07-01/relay.json

```diff
@@ -1991,8 +2109,21 @@
         }
       },
       "description": "Parameters supplied to the Regenerate Auth Rule."
+    },
+    "ErrorResponse": {
```

Comment:
What is the schema of the Error Response?
Is it `{ "error": { code: "Some code", "message": "Some meaningful message" } }` OR `{ code: "Some code", "message": "Some meaningful message" }` ?